plutus-benchmark/plutus-benchmark.cabal

```diff
@@ -357,3 +357,41 @@ executable ed25519-throughput
     , plutus-core           ^>=1.1
     , plutus-tx             ^>=1.1
     , plutus-tx-plugin      ^>=1.1
+
+---------------- script contexts ----------------
+
+library script-contexts-internal
+  import:          lang
+
+  if impl(ghc <9.0)
```

Comment:
Why do we have this now?  I can't run these locally because of this but if I change it I get errors because other things also have this requirement.